stylesheets/components/Cards/Cards.css

```diff
@@ -0,0 +1,214 @@
+
+    /* Card CSS */
+    .card {
+        position: relative;
+        min-width: 10.625rem;
+        max-width: 35rem;
+        min-height: 9.375rem;
+        max-height: 400px;
+        background-color: white;
+        border-radius: 12px;
+        box-shadow: var(--box-shadow);
+    }
+
+    .card--small {
+        width: min-content;
+        height: min-content;
+    }
+
+    .card__img--small {
+        width: 10.625rem;
+        height: 10.625rem;
+    }
+
+    .card__img--small img {
+        border-radius: 12px;
+    }
+
+    .card__content--column {
+        display: flex;
+        flex-direction: column;
+        padding: 10px 12px;
+    }
+
+    .card__content--row {
+        display: flex;
+        flex-direction: row;
+        padding: 10px 12px;
+    }
+
+    .card__icon {
+        position: absolute;
+        outline: 0px;
+        top: 8px;
+        right: 8px;
+        background-color: rgba(230, 230, 230, 0.5);
+        font-size: 1rem;
+    }
+
+    .card__icon:hover {
+        position: absolute;
+        top: 8px;
+        right: 8px;
+        color: var(--component-red-05);
+        background-color: var(--component-red-01);
+        font-size: 1rem;
+    }
+
+    /* Mini Card */
+    .card--mini {
+        width: min-content;
+        height: max-content;
+    }
+
+    .card__img--mini img {
+        border-radius: 12px;
+    }
+
+    .mini--content {
+        position: absolute;
+        width: 100%;
+        bottom: 0px;
+        background: rgba(197, 220, 250, 0.5);
+        border-radius: 0px 0px 12px 12px;
+    }
+
+    .card__tag {
+        position: absolute;
+        top: 8px;
+        left: 8px;
+        padding: 2px 6px;
+        border-radius: 4px;
+        background: var(--component-yellow-03);
+        ;
```

Comment:
remove this `;`

Reply:
Resolved in the latest commit. Thank you.